website/docs/r/dataproc_job.html.markdown

```diff
@@ -0,0 +1,364 @@
+---
+layout: "google"
+page_title: "Google: google_dataproc_job"
+sidebar_current: "docs-google-dataproc-job"
+description: |-
+  Manages a job resource within a Dataproc cluster.
+---
+
+# google\_dataproc\_job
+
+Manages a job resource within a Dataproc cluster within GCE. For more information see
+[the official dataproc documentation](https://cloud.google.com/dataproc/).
+
+!> **Note:** This resource does not support 'update' and changing any attributes will cause the 
```

Comment:
how does "changing any attributes will cause the resource to be recreated" sound?